Plugins/AWSLambdaPackager/Plugin.swift

```diff
@@ -67,6 +67,7 @@ struct AWSLambdaPackager: CommandPlugin {
 
         // create the archive
         let archives = try self.package(
+            packageName: context.package.displayName,
```

Comment:
Wasn't sure if there was a better way to get the package name when creating the resources directory name.

Reply:
I think this is the only way to retrieve a package name. 
See https://github.com/swiftlang/swift-package-manager/blob/b24cc8b547f952ee4a954db939f115a26946fe2b/Sources/PackagePlugin/PackageModel.swift#L16

The other option is to use the directory name but it's not necessarily the same.